Rewrite of chapter ledes (#375)
worldbank/d4di: chapters/publication.tex

fussing with the technical requirements of publication.
It is in nobody's interest for a skilled and busy researcher
to spend days re-numbering references (and it can take days)
when a small amount of up-front effort can automate the task.
In this section we suggest several methods --
collectively referred to as dynamic documents --
for managing the process of collaboration on any technical product.

For most research projects, completing a manuscript is not the end of the task.
Academic journals increasingly require submission of a replication package,
which contains the code and materials needed to create the results.
These represent an intellectual contribution in their own right,
because they enable others to learn from your process
and better understand the results you have obtained.
Holding code and data to the same standards a written work

> is a new practice for many researchers.

Reviewer: This part is repeated from the introduction of the lede. Something is off here, probably a copy and paste error

>For most research projects, completing a manuscript is not the end of the task.
Academic journals increasingly require submission of a replication package,
which contains the code and materials needed to create the results.
These represent an intellectual contribution in their own right,
because they enable others to learn from your process
and better understand the results you have obtained.
Holding code and data to the same standards a written work
is a new practice for many researchers.

Reviewer 2: Fixed in 22ab8f485d55dc97dd1ec6a1b7577c1ae8ec47d

Hence, this step represents the conclusion of the system
of transparent, reproducible, and credible research we introduced
from the very first chapter of this book.

> In this chapter, we first discuss tools and workflows for collaborating on technical writing.

Reviewer 3: ```suggestion
We start the chapter with a discussion about tools and workflows for collaborating on technical writing.
```

(The paragraph already starts with "in this chapter")
